conference.js

```diff
@@ -1990,7 +1990,8 @@ export default {
             }
         );
 
-        APP.UI.addListener(UIEvents.EMAIL_CHANGED, this.changeLocalEmail);
+        APP.UI.addListener(UIEvents.EMAIL_CHANGED,
+            this.changeLocalEmail.bind(this));
```

Comment:
Just out of curiosity, do we need the bind?

Reply:
We don't need bind.
But since all other functions are using it, it was more like a style fix.

Reply:
https://github.com/AkshitKrNagpal/jitsi-meet/blob/59f2f0143fb409d14ddac2e1b6bccf8d54d57cae/conference.js#L2026

Reply:
Like this.
https://github.com/jitsi/jitsi-meet/blob/361e5f0fad58c930a1c7825b1add6db14e75c85b/conference.js#L2024-L2025